elements/bulbs-video/components/revealed.js

```diff
@@ -43,6 +16,31 @@ function makeGaPrefix () {
 
 export default class Revealed extends React.Component {
   componentDidMount () {
+    /*
```

Comment:
In some environment, it escapes me now whether it was a test, dev, or deployed environment this wasn't working properly unless it was moved to a late binding such as this.

This is on the way out with @spra85's JWPlayer #82 branch, so I wouldn't fuss to much about it here.